docs/extensions/cross-chain-data/example_pt_1.mdx

```diff
@@ -0,0 +1,273 @@
+---
+sidebar_position: 4
+sidebar_label: "Example Pt. 1: Kwil Account Credits"
+id: oracle-example-part-1
+title: "Example Pt. 1: Kwil Account Credits"
+description: "Create a resolution that credits a Kwil account with the chain's native gas token."
+slug: /extensions/oracle-example-part-1
+---
+
+In this tutorial, we will create a resolution that can credit accounts with the networks native gas token. This example is based off of the
+`credit_account` resolution found in the default Kwil implementation. It is purely for example purposes, since the default implementation
+is already available in the Kwil network. The default implementation can be found [here](<https://github.com/kwilteam/kwil-db/blob/main/extensions/resolutions/credit/credit.go>).
```

Comment:
```suggestion
In this tutorial, we will create a resolution that can credit accounts with the networks native gas token. This example is based off of the
`credit_account` resolution found in the [default Kwil implementation](https://github.com/kwilteam/kwil-db/blob/main/extensions/resolutions/credit/credit.go). It is purely for example purposes, since the default implementation
is already available in the Kwil network.
```

move hyperlink to where it is referenced.